cpp/src/arrow/testing/random.h

```diff
@@ -52,7 +60,7 @@ class ARROW_TESTING_EXPORT RandomArrayGenerator {
   std::shared_ptr<Array> Boolean(int64_t size, double true_probability,
                                  double null_probability = 0);
 
-  /// \brief Generates a random UInt8Array
+  /// \brief Generate a random UInt8Array
```

Comment:
nit: why the change?  If we want to standardize on one vs the other we should probably update the style guide?

Reply:
Hmm, I can add this somewhere in the docs. Most of our docstrings use infinite, AFAIK.